### PR TITLE
register cybertron is a dev domain

### PR DESCRIPTION
**Pull request by infy11**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/cybertron.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "My personal devbox for personal project",
+    "owner": {
+      "username": "varnitgoyal95",
+      "email": "varnitgoyal95@gmail.com"
+    },
+    "record": {
+      "A": ["140.238.250.204"]
```
Comment (Stef-00012) [inline domains/cybertron.json:8]: I cannot reach ur website

> This site can’t be reachedhttp://140.238.250.204/ is unreachable.
ERR_ADDRESS_UNREACHABLE

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/cybertron.json (hunk 1 above)
Comment (infy11) [reply domains/cybertron.json:8]: its on different port try to access 

140.238.250.204:5601

Comment (CuteDog5695) [reply domains/cybertron.json:8]: Is this domain for your AI program?

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by infy11** [comment]
Yes

**Pull request closed without merging**